apps/checklist/1.0.0/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Checklist",
+    "version": "1.0.0",
+    "description": "Simple checklist",
+    "parameters": [
+        {
+            "name": "name",
```

Comment:
Name is not needed as the app name is the checklist name

I think we should pass some first checlist items optionally as the arg